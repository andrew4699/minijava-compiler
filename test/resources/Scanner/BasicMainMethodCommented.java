public class BasicMainMethod { // this class is pretty cool
  public static void main(String[] args) { // this defines a new method
    System.out.println(777); // this prints out the // number 777
    // ooo we're about to close the method
  } // closing the method
} // this class is done
