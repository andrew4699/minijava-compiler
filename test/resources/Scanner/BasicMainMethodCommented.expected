PUBLIC CLASS ID(BasicMainMethod) LBRACE
  PUBLIC STATIC VOID ID(main) LPAREN ID(String) LBRACKET RBRACKET ID(args) RPAREN LBRACE
    DISPLAY LPAREN INT(777) RPAREN SEMICOLON
  RBRACE
RBRACE
